### DISCUSSION TITLE
Down arrow keybind no longer works

### DISCUSSION THREAD

**Discussion by DragonZV** [marlonrichert/zsh-autocomplete#395] [Q&A] [answered]
### Environment

```zsh
print $VENDOR $OSTYPE $SHELL $ZSH_ARGZERO $ZSH_PATCHLEVEL
print -l $_autocomplete__funcfiletrace
git -C ~zsh-autocomplete log --oneline -n1
```
```
pc linux-gnu /bin/zsh zsh zsh-5.8-0-g77d203f
/home/arch/.znap/zsh-snap/functions/.znap.source:26
/home/arch/.znap/zsh-snap/functions/znap:42
/home/arch/.zshrc:16
```

### Steps to reproduce

The error doesn't occur in the mini shell you provided for some reason, which is what confuses me.
My config can be found [here](https://sourceb.in/8tFx45NKoA).

**Comment by marlonrichert** [comment, marked as answer]
> The error doesn't occur in the mini shell you provided for some reason, which is what confuses me.

This means that the problem is in your config and not in Zsh Autocomplete. Delete lines 22 - 105 and you should be fine.

**Reply by DragonZV** [reply]
Thanks!
